apertium-eng-spa.eng-spa.dix

```diff
@@ -38983,8 +38983,278 @@
     <e><p><l>fire<b/>brigade<s n="n"/></l><r>brigada<g><b/>de<b/>bomberos</g><s n="n"/><s n="f"/></r></p></e>
     <e><p><l>spear<s n="n"/></l><r>lanza<s n="n"/><s n="f"/></r></p></e>
     <e><p><l>eye<b/>shadow<s n="n"/></l><r>sombra<g><b/>de<b/>ojos</g><s n="n"/><s n="f"/></r></p></e>
+
+    <e a="albertonl"><i>Bennet<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Bingley<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Lizzy<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Kitty<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Hurst<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Darcy<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Phillips<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Lucases<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Forster<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Louisa<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Gardiner<s n="np"/><s n="ant"/></i></e>
+    <e a="albertonl"><i>Hertfordshire<s n="np"/><s n="loc"/></i></e>
+    <e a="albertonl"><i>Derbyshire<s n="np"/><s n="loc"/></i></e>
+    <e a="albertonl"><i>Longbourn<s n="np"/><s n="loc"/></i></e>
+    <e a="albertonl"><i>Cheapside<s n="np"/><s n="loc"/></i></e>
+    <e a="albertonl"><i>Phillips<s n="np"/><s n="al"/></i></e>
     <e a="albertonl" r="LR"><p><l>over<s n="pr"/></l><r>a<b/>lo<b/>largo<b/>de<s n="pr"/></r></p></e>
     <e a="albertonl" r="LR"><p><l>throughout<s n="pr"/></l><r>a<b/>lo<b/>largo<b/>de<s n="pr"/></r></p></e>
+    <e a="albertonl"><p><l>rightful<s n="adj"/></l>         <r>legítimo<s n="adj"/></r></p></e>
+    <e a="albertonl" r="LR"><p><l>tiresome<s n="adj"/></l>  <r>irritante<s n="adj"/></r></p></e>
+    <e a="albertonl"><p><l>scrupulous<s n="adj"/></l>       <r>escrupuloso<s n="adj"/></r></p></e>
+    <e a="albertonl"><p><l>hearty<s n="adj"/></l>           <r>caluroso<s n="adj"/></r></p></e>
+    <e a="albertonl"><p><l>humourless<s n="adj"/></l>       <r>sin<b/>sentido<b/>del<b/>humor<s n="adj"/></r></p></e>
```

Comment:
This is going to result in not great translations.

Reply:
".... not great ...."
What is the actual problem please?
( I would like to know how to add words to the dictionaries)

Reply:
Maybe last line (as its an adjective) should just be 
`<r>sin<b/>humor<s`  .... ?